languagetool-language-modules/sr/src/main/java/org/languagetool/language/BosniaSerbian.java

```diff
@@ -0,0 +1,15 @@
+package org.languagetool.language;
```

Comment:
I know these files are almost trivial, but could you add a copyright/license header anyway?

Reply:
Will do.